CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.5.2-rc1
+### Note
+- Helm charts are changed to enforce PodDisruptionBudget for multi-replica clusters and PDB is removed from install manifests.
```

Comment:
Does the user need to remove PDB manually if running a single replica?

Reply:
PDB is not created for a single replica but in the case of multiple replicas, the user would need to remove PDB manually.